scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityCreationTemplate.java

```diff
@@ -250,10 +260,15 @@ private Optional<SchedulingActivityDirective> createInstanceForReal(final String
         throw new UnsupportedOperationException("Activity creation duration arguments cannot depend on simulation results.", e);
       }
       duration.ifPresent(d -> tnw.addDurationInterval(name, d, d));
+      storedDur = duration.get();
     }
     final var success = tnw.solveConstraints();
     if (!success) {
-      logger.warn("Inconsistent temporal constraints, will try next opportunity for activity placement if it exists");
+      logger.warn("Inconsistent static temporal constraints, will try next opportunity for activity placement if it exists");
```

Comment:
Minor, can ignore: There is already a log message before this method is called, so this log statement doesn't need to comment on what its call location may do and should instead comment on what it failed to do.
```suggestion
      logger.warn("Inconsistent static temporal constraints, cannot place activity in interval");
```